InternTrack.Core.Api.Tests.Acceptance/Usings.cs

```diff
@@ -0,0 +1 @@
+using Xunit;
```

Comment:
Remove this it is not needed. All usings are placed at the top each file by the coder. Global using are considered magic code and unacceptable in The Standard principles.